building/build-iso/utils/fetch-package-from-repo.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/python3.5
+
+import sys
+import subprocess
+import urllib.parse
+import os
+import hashlib
+import argparse
+
+repo_url = "http://web.mit.edu/hyades/debian/"
+
+argparser = argparse.ArgumentParser(description='Download a package from Homeworld repo')
+argparser.add_argument('package_name', help="the name of the package to fetch")
+argparser.add_argument('--verbose', '-v', action="store_true", help="make the output verbose")
+args = argparser.parse_args()
+verbose = args.verbose
+package_name = args.package_name
```

Comment:
It might be worth refactoring these two scripts so that they have better defined interfaces -- currently, the interfaces involve plopping down files in hardcoded locations relative to the current directory, which works except that this kind of style makes it harder to figure out what's going on overall without reading all of the moving parts.